README.md

```diff
@@ -1,3 +1,91 @@
 # clue
 
-Experimental GraphQL client.
\ No newline at end of file
+Experimental GraphQL client.
+
+## Usage
+
+### 1) Create a client
+
+Either:
+  * A `GraphQLClient[F[_], S]` (supporting queries and mutations), or
+  * A `GraphQLStreamingClient[F[_], S]` (supporting queries, mutations and subscriptions).
+
+  `S` is a type denoting the schema. It can be any type, even a phantom type. It's only used to type-match clients and operations.
```

Comment:
It could be the case if we declare schema types inside an `object`.

In the example I use a `sealed trait` (which I see now I've incorrectly written as `sealed type`(???) ) just because `type` is not supported stand-alone (it must be within another definition), but whenever possible, using a `type` is preferred since it won't generate a class.